src/vuex/index.ts

```diff
@@ -1,7 +1,10 @@
 import Vuex from 'vuex';
+import Vue from 'vue';
 
 import app from './modules/app';
 
+Vue.use(Vuex);
+
```

Comment:
Forgot to actually add Vuex to the Vue instance. Can't be done in main as it must occur before we create the store